engine/src/main/resources/assets/i18n/menu_fa.lang

```diff
@@ -0,0 +1,244 @@
+    "bloom-effect": "بلووم                        ⁠⁠",
+    "bobbing": "حرکت به بالا و پایین",
+    "camera-blur": "تیرگی دوربین",
+    "camera-blur-max": "تیرگی دوربین در بالاترین حد",
+    "camera-blur-normal": "تیرگی دوربین در حد طبیعی",
+    "camera-blur-off": "خاموش کردن تیرگی دوربین",
+    "camera-blur-some": "Some",
+    "camera-details-settings": "تنظیمات جزییات گرافیکی دوربین",
+    "camera-film-grain": "بافت فیلم",
+    "camera-setting": "تنظیمات دوربین",
+    "camera-setting-cinematic": "سینمایی",
+    "camera-setting-normal": "تنظیمات معمولی دوربین",
+    "camera-setting-smooth": "نرم",
+    "category-behavior": "دسته بندی عمومی رفتار",
+    "category-general": "دسته بندی عمومی رفتار",
+    "category-interaction": "اثر متقابل",
+    "category-inventory": "دسته بندی -بازبینی",
+    "category-mouse": "موش واره",
+    "category-movement": "حرکت",
+    "category-nui": "NUI",
+    "change-keybind-popup-message": "is already bound to an action. Rebind anyway?",
+    "change-keybind-popup-title": "کلید هم اکنون مهیا است",
+    "clamp-lighting": "مهار نور",
+    "cloud-shadows": "سایه های ابر",
+    "connecting-to": "Connecting to",
+    "connection-failed": "قطع اتصال",
+    "could-not-connect-to-server": "نمی تواند به کارساز ( سرور) وصل شود",
+    "credits": "اعتبارها",
+    "create-game": "خلق",
+    "create-game-title": "خلق بازی",
+    "create-preview": "پیش دید کردن",
+    "custom-servers": "بر حسب عادت",
+    "deactivate-module": "غیر فعال",
+    "death-screen-title": "شما مرده اید",
+    "default-binding": "از بین رفتن اتصال",
+    "delete-game": "حذف کردن",
+    "dev-tools-bt-editor": "نمایشگر درختی ویرایش رفتاری",
+    "dev-tools-nui-editor": "ویرایشگر صفحه ان. یو. ال",
+    "dev-tools-skin-editor": "ویرایشگر اسکین ان.یو. ال",
+    "dev-tools-title": "ابزار توسعه دهنده",
+    "dialog-cancel": "لغو کردن",
+    "dialog-ok": "درست است",
+    "disable-all-modules": "از کار انداختن همه",
+    "download-module": "دانلود",
+    "downloading-server-list": "دانلود لیست کارساز..",
+    "drop-item": "قطع مطلب",
+    "edit-server": "ویرایش",
+    "edit-server-title": "اضافه/ویرایش  کارساز (سرور)",
+    "enter-username-message": "Hello and welcome to Terasology's multiplayer!\nWould you like to use a non-default username?",
+    "enum-editor-prompt": "انتخاب مقدار برای منحنی",
+    "enum-editor-title": "ویرایشگر منحنی داده ها",
+    "error-downloading-server-list": "اشتباه در دانلود فهرست کترساز (سرور)",
+    "exit-game": "خروج از بازی تراسولوژی",
+    "extra-lighting": "پرتوهای اضافه",
+    "eye-adaption": "انطباق چشمی",
+    "failed-to-join": "عدم موفقیت در پیوستن",
+    "flickering-lights": "نورهای لرزان",
+    "fog": "مه",
+    "fov-amount": "میزان میدان دید",
+    "framerate-limit": "محدودیت سازه",
+    "game-name": "اسم بازی",
+    "game-settings": "تنظیمات",
+    "game-worlds": "دنیاها",
+    "gameplay": "محتویات نمایش بازی",
+    "gameplay-info": "can click 'modules' to customize further",
```

Comment:
This is untranslated - can you wrap this up just so the language is 100% finished? [5]